test/Concurrency/Runtime/async_taskgroup_throw_rethrow.swift

```diff
@@ -7,8 +7,6 @@
 // UNSUPPORTED: use_os_stdlib
 // UNSUPPORTED: back_deployment_runtime
 
-// XFAIL: OS=windows-msvc
```

Comment:
@benlangmuir was about to make it unsupported but seems we'd rather make it working now; 
Please sanity check https://github.com/apple/swift/pull/37544/files

Reply:
@compnerd when should it have started passing? I was still seeing roughly 50-50 xpass/xfail on the Windows PR bot today.  Just want to make sure it's fully working.

Reply:
@benlangmuir it would have been some time today (the change was merged today)

Reply:
Okay, I didn't check exhaustively, but I think it's all been XPASS since that llvm change was merged, so thanks!